content/docs/refs-and-the-dom.md

```diff
@@ -11,7 +11,7 @@ redirect_from:
   - "tips/children-undefined.html"
 ---
 
-Refs provide a way to access DOM nodes or React elements created in the render method.
+Refs provide a way to access HTML elements or React elements created in the render method.
```

Comment:
But now it's confusing since HTML elements does not include SVG elements. I'm not even sure what the "React elements" refers to, React component instances?